src/main/java/net/snowflake/client/jdbc/SnowflakeResultSetSerializableV1.java

```diff
@@ -143,6 +143,7 @@ public String toString() {
   boolean treatNTZAsUTC;
   boolean formatDateWithTimezone;
   boolean useSessionTimezone;
+  int sessionClientMemoryLimit;
```

Comment:
curious what this is for?

Reply:
There was a possibility of a nullptr exception if this parameter is null. Although, probably this is not the only parameter where that is the case. But this is where I noticed it.